plugins/repository-hdfs/build.gradle

```diff
@@ -91,6 +91,13 @@ dependencies {
   if (isEclipse == false) {
     testRuntimeOnly files(project(':test:fixtures:krb5kdc-fixture').ext.krb5Keytabs("hdfs", "hdfs_hdfs.build.opensearch.org.keytab").parent)
   }
+
+  configurations.all {
```

Comment:
Why do we need that if line 77 already uses this version?

Reply:
The transitive dependency (via hadoop) was showing up in thirdpartyaudit dependencies.   I think I could also use `strictly` on the line 77 dependency?

Reply:
> The transitive dependency (via hadoop) was showing up in thirdpartyaudit dependencies. I think I could also use `strictly` on the line 77 dependency?

Or exclude it from `hadoop`-related artifacts?